qa/tasks/nfs_ganesha_rgw_v2.py

```diff
@@ -160,11 +161,25 @@ def task(ctx, config):
 
     # run the test
 
+    remote.run(
```

Comment:
what is the use of this, this part was not there before i think.

Reply:
Removed unwanted code now